packages/manager/apps/sms/package.json

```diff
@@ -63,4 +64,4 @@
     "ovh-manager-webfont": "^1.2.0",
     "webpack-merge": "^4.2.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
No need to add `angular-ui-bootsrap` too ?

Reply:
done on last commit, my bad :)